README.md

```diff
@@ -35,6 +35,7 @@ There are three kinds of version labels you can use.
 - `packages` - Space delimited list of packages to install.
 - `version` - Version of cache to load. Each version will have its own cache. Note, all characters except spaces are allowed.
 - `execute_install_scripts` - Execute Debian package pre and post install script upon restore. See [Caveats / Non-file Dependencies](#non-file-dependencies) for more information.
+- `if-empty-packages` - Desired behavior when the given `packages` is empty. `'error'` (default), `'warn'` or `'ignore'`.
```

Comment:
Great documentation. If you could update though to use underscores so it is consistent.

Bit of a nit. Command line params are typically nouns. empty_packages_behavior?

Reply:
Fine. Renamed to `empty_packages_behavior`.